src/main/resources/assets/react/documentTypes/NewOrEditDocumentType.jsx

```diff
@@ -182,6 +183,10 @@ export function NewOrEditDocumentType({
 								properties
 							});*/
 							setId(_id);
+							setParentState({
```

Comment:
Do you know how React useState works. What you are doing here is to overwrite the whole state, not just setting _id and _name. In other works collectionsArr, interfacesArr and open becomes undefined. That might not be what you wanted?

https://reactjs.org/docs/hooks-reference.html#functional-updates

Unlike the setState method found in class components, useState does not automatically merge update objects.

Reply:
Updated based on the feedback